app/scripts/loadout/dimLoadout.directive.js

```diff
@@ -3,9 +3,10 @@
 
   angular.module('dimApp').directive('dimLoadout', Loadout);
 
-  Loadout.$inject = ['dimLoadoutService'];
+  Loadout.$inject = ['dimLoadoutService','dimSettingsService'];
 
   function Loadout(dimLoadoutService) {
```

Comment:
Thanks for adding it as a toggle. Love it-- though, I'm getting a console error, this might actually need to be 

`function Loadout(dimLoadoutService, dimSettingsService) {`